test/test-framework/test/e2e/memcached_test.go

```diff
@@ -45,8 +45,8 @@ func TestMemcached(t *testing.T) {
 	}
 	// run subtests
 	t.Run("memcached-group", func(t *testing.T) {
-		t.Run("Cluster", MemcachedCluster)
-		t.Run("Cluster2", MemcachedCluster)
+		t.Run("AVeryLongClusterNameThatIsSoLong", MemcachedCluster)
```

Comment:
Why are these names changed?

Reply:
I think he tried to demonstrate that the issue was solved. 
However, I agree with @estroz. I would stick with the previous names either.

Reply:
Yep, that's why I did it. Before this change that test would fail. I'll change it back.